.travis.yml

```diff
@@ -1,12 +1,15 @@
 sudo: required
 dist: trusty
 
+arch:
+  - amd64
+  - arm64
+
 language: go
 go:
   - 1.10.x
 go_import_path: github.com/GoogleContainerTools/container-structure-test
 
 script:
   - ./hack/.gofmt.sh
-  - make
```

Comment:
I moved it into the `tests/structure_test_tests.sh:38` because that's where we were making all the binaries anyway. 

See before we were doing:
- make (outside of the script)
- make cross (inside the script)
- make image (inside the script)

So to keep things consistent I moved the `make` into the script also.
I can move it back out if you prefer?

Reply:
ahh right sorry now I see what you did. the end result looks the same so seems fine to me